codebuild/spec/buildspec_unit_coverage.yml

```diff
@@ -0,0 +1,43 @@
+---
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+# this file except in compliance with the License. A copy of the License is
+# located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
+version: 0.2
+
+env:
+  parameter-store:
+    CODECOV_TOKEN : codecov-upload-token
+  variables:
+    # CODEBUILD_ is a reserved namespace.
+    CB_BIN_DIR: "./codebuild/bin"
+    CC: "/usr/bin/clang"
+    CXX: "/usr/bin/clang++"
+
+phases:
+  pre_build:
+    commands:
+      - |
+        if [ -d "third-party-src" ]; then
+          cd third-party-src;
+        fi
+      - ln -s /usr/local $CODEBUILD_SRC_DIR/test-deps
```

Comment:
This should probably be part of the if "third-party-src" block; e.g. `ln -s /usr/local/ $CODEBUILD_SRC_DIR/third-party-src/test-deps` like [here](https://github.com/aws/s2n-tls/blob/main/codebuild/spec/buildspec_ubuntu.yml#L27)

Reply:
In the example you linked it looks like we are linking against a different directory path - `$CODEBUILD_SRC_DIR/third-party-src/test-deps` vs `$CODEBUILD_SRC_DIR/test-deps`. Is that the one that I should be using?

Do we have any documentation about when we should be using those different paths? 

Also, now that I'm looking at this I don't think I need that third-party-src block at all. Any objections to me just removing it?

Reply:
Discussed offline, that's for compatibility with some legacy build systems.